src/Components/WebAssembly/Samples/HostedBlazorWebassemblyApp/Client/wwwroot/css/open-iconic/FONT-LICENSE

```diff
@@ -0,0 +1,86 @@
+SIL OPEN FONT LICENSE Version 1.1
```

Comment:
Could we have a more minimal app here? We don't really need a full-fledged app in the sandbox.

Reply:
@pranavkm I chose to use the full app because otherwise in many cases is much more painful to try stuff with something that doesn't look like a template app and we only have a testasset that is close to this which is the auth project, and it's a pain to modify that one to try stuff.

Reply:
From skim reading this, it looks like the stock project template, and the app doesn't contain anything specifically to do with state preservation. So arguably it's orthogonal to this PR. Is that correct?

If so that seems OK to me. While I sympathise with @pranavkm's point, as long as this is a pure "sandbox" that isn't coupled to any particular optional features (and isn't used from E2E tests) then it seems potentially useful to have it in the solution. We *could* simplify it a bit by removing all the 3rd-party CSS stuff, but I wouldn't say that's a priority.

If it was for a test, I'd certainly argue it should be as minimal as possible to exercise the feature being tested.

Reply:
Yes, I did modify the stock template a bit to make it easier to do prerendering, but that's it. It's meant for us to be able to try stuff quickly. Otherwise it takes me 5-10 minutes every time I want to try something in this space. We also have a BlazorServerApp project in the samples for that same reason.

Reply:
We can always decide to remove it in the future if we don't want it any longer